### DISCUSSION TITLE
Edit readme.md file

### DISCUSSION THREAD

**Discussion by NiklasPeterson** [community/community#3033] [Mobile] [answered]
It would be awesome if you where able to edit readme file :)

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @NiklasPeterson! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Comment by edward-tobilko** [comment]
+1

**Comment by akashdip2001** [comment]
Of course! I can help you edit your README file. Please provide the content or specific changes you'd like, and we can work on it together.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |
